### PR TITLE
Screenshots and updates

### PR DESCRIPTION
**Pull request by gilgongo**
Fixes:

https://github.com/jamulussoftware/jamulus/pull/2640 - sound alert 
https://github.com/jamulussoftware/jamulus/pull/2199 - screenshot (also 2424)

(I looked at https://github.com/jamulussoftware/jamulus/pull/2673 but the website seems fine)

### CODE REVIEW

**Review by ann0see — APPROVED** [review]

File: wiki/en/Software-Manual.md [hunk 1]
```diff
@@ -146,6 +146,10 @@ This changes the visual style of the audio meters, independently of the chosen s
 
 This sets the number of rows displayed in the server audio mixer, for use with larger ensembles.
 
+### Audio Alerts
+
+This turns on a sound alert for when someone joins a chat, or when receiving a new chat message.
```
Comment (ann0see) [inline wiki/en/Software-Manual.md]: It also gives a message if someone joins and doesn't send a chat

Comment (gilgongo) [reply wiki/en/Software-Manual.md]: I think that's implied, no?

Comment (ignotus666) [reply wiki/en/Software-Manual.md]: I think he means that there's an alert when someone joins a server, not a chat. Then there's another alert for new chat messages.

Comment (gilgongo) [reply wiki/en/Software-Manual.md]: Oh - I didn't realise the audio alert was for server joining too. I better amend that wording then.

Comment (gilgongo) [reply wiki/en/Software-Manual.md]: So the audio alert plays when EITHER somebody joins a Server, OR when somebody sends a new chat message, is that right?

Comment (ignotus666) [reply wiki/en/Software-Manual.md]: As far as I know, yes, with a different sound for each event.

**Review by ann0see — APPROVED** [review]

**Pull request merged**